File: LeetCode/Simulation/Golang/Leetcode1374-generate-a-string-with-characters-that-have-odd-counts.go
package sbgo

func generateTheString(n int) string {
    if n % 2 == 0 {
        return strings.Repeat("a",n - 1) + "b"
    }
    if n == 1 {
        return "a"
    }
    return strings.Repeat("a",n - 2) + "b" + "c"
}

File: LeetCode/Simulation/Python/Leetcode1374-generate-a-string-with-characters-that-have-odd-counts.py
class Solution:
    def generateTheString(self, n: int) -> str:
        if n % 2 == 0:
            return 'a' * (n - 1) + 'b'
        if n == 1: return 'a'
        return 'a' * (n - 2) + 'b' + 'c'
